classes/local/upload_helper.php

```diff
@@ -76,10 +76,11 @@ public static function get_upload_jobs($courseid) {
 
         $allnamefields = get_all_user_name_fields(true, 'u');
 
-        $sql = "SELECT uj.*, f.filename, f.filesize, $allnamefields
+        $sql = "SELECT uj.*, f.filename, f.filesize, $allnamefields, md.metadata
                 FROM {block_opencast_uploadjob} uj
-                JOIN {files} f ON uj.fileid = f.id AND f.component = :component
+                JOIN {files} f ON uj.presenter_fileid = f.id OR uj.presentation_fileid = f.id AND f.component = :component
                 JOIN {user} u ON f.userid = u.id
+                JOIN {block_opencast_metadata} md ON uj.id = md.uploadjobid 
                 AND f.filearea = :filearea AND f.filename <> '.'
```

Comment:
By resolving the upper request, please move this line also to each of the JOINS. I have no idea, why this got detached from the ON statement for the files-JOIN.